tensorboard/webapp/runs/views/runs_table/runs_table_component.scss

```diff
@@ -19,7 +19,6 @@ $_table-content-row-min-height: 43px;
 $_font-size: 13px;
 
 :host {
-  background-color: #fff;
```

Comment:
Looks like this change affects light-theme.  Let's preserve light-theme's run table colors:

![image](https://user-images.githubusercontent.com/2322480/120756164-c5bb8600-c4c3-11eb-8017-c8bd60c7244e.png)